src/slurm_plugin/clustermgtd.py

```diff
@@ -107,7 +108,11 @@ def get_status(self, fallback=None):
     def update_status(self, current_status, next_status):
         try:
             self._table.put_item(
-                Item={"Id": self.COMPUTE_FLEET_STATUS_KEY, self.COMPUTE_FLEET_STATUS_ATTRIBUTE: str(next_status)},
+                Item={
+                    "Id": self.COMPUTE_FLEET_STATUS_KEY,
+                    self.COMPUTE_FLEET_STATUS_ATTRIBUTE: str(next_status),
+                    self.LAST_UPDATED_TIME_ATTRIBUTE: str(datetime.now()),
```

Comment:
let's always compute the datetime in UTC: `datetime.now(tz=timezone.utc)`

Reply:
Done